sdk-autoconfigure-support/src/test/java/io/opentelemetry/instrumentation/internal/resources/ResourceProviderPropertiesCustomizerTest.java

```diff
@@ -24,8 +24,16 @@
 import org.junit.jupiter.api.DynamicTest;
 import org.junit.jupiter.api.TestFactory;
 
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
```

Comment:
i dont think these comments are needed for tests ([source](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/custom-checks/src/main/java/io/opentelemetry/javaagent/customchecks/OtelInternalJavadoc.java#L52-L54)). Same thing for `AddThreadDetailsSpanProcessorTest`

Reply:
Hopefully resolved with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/14281